sass/variables/_colors.scss

```diff
@@ -1,3 +1,10 @@
+/* Colors for Contact-us & About-us pages */
+$color__background-contact-about: #2fbeb6;
+$color__text-title-contact-about: #FFF;
+$color__text-content-contact-about: #404041;
```

Comment:
I'd use camelCase for these variables.
Kind of questioning BEM btw now. Think it's a decent convention but preferring SMACCs